sourcegraph/sourcegraph.proto

```diff
@@ -142,7 +146,9 @@ message Repo {
 	// Fork is whether this repository is a fork.
 	bool fork = 12;
 
-	// Mirror is whether this repository is a mirror.
+	// Mirror indicates whether this repo's canonical location is on
+	// another server. Mirror repos track their upstream and are not
+	// eligible for discovery on this server.
 	bool mirror = 13;
```

Comment:
Is it a true statement that any *mirrored* repository is not hosted on a Sourcegraph instance. I.e., mirrored repositories mirror only repositories hosted on other services (GitHub, Bitbucket, etc.)?